<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>地面报文译码</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>641</width>
     <height>341</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>651</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>561</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>当前文件：</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>152</width>
     <height>31</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="currentText">
    <string>0</string>
   </property>
   <property name="maxVisibleItems">
    <number>8</number>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>台站号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>日期：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dateEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>Form</receiver>
   <slot>updateDate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>updateTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>showResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>724</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>updateStation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>Form</receiver>
   <slot>updateStation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showText()</slot>
  <slot>updateDate()</slot>
  <slot>updateTime()</slot>
  <slot>showResult()</slot>
  <slot>updateStation()</slot>
 </slots>
</ui>
